packages/graph/Makefile.toml

```diff
@@ -23,13 +8,25 @@ args = ["compose", "--env-file", "${REPO_ROOT}/packages/graph/.env", "up", "migr
 cwd = "${REPO_ROOT}/packages/graph/deployment"
 
 [tasks.deployment-up]
+run_task = { name = ["docker-compose-deployment-up", "migrate-db"] }
+
+[tasks.docker-compose-deployment-up]
 private = false
 category = "Deploy"
 description = "Spins up the deployment environment"
 extend = "docker"
 args = ["compose", "--env-file", "../.env", "up", "--wait", "--build"]
 cwd = "${REPO_ROOT}/packages/graph/deployment"
 
+[tasks.migrate-db]
+private = false
+category = "Deploy"
+extend = "run-task"
+description = "Run DB migrations using the Graph binary"
+# This isn't perfect, but it gives us the possbility to run the `migrate-db` task from either this
+# `Makefile.toml` or the one in the `./hash_graph` directory.
+args = ["run", "--manifest-path", "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/../../../packages/graph/hash_graph/Cargo.toml", "--", "migrate"]
```

Comment:
Is `${REPO_ROOT}` not working? I think that would be easier to read.

Reply:
I addressed this with the CWD change in 0c40fda7593f4a3b5ebdf84d33147b9a8776b65b :)